containers/cuda/Containerfile

```diff
@@ -179,12 +180,15 @@ ENV C_INCLUDE_PATH="${INTEL_MKL_HOME}/include:${C_INCLUDE_PATH}"
 ENV CPLUS_INCLUDE_PATH="${INTEL_MKL_HOME}/include:${CPLUS_INCLUDE_PATH}"
 ENV CMAKE_INCLUDE_PATH="${INTEL_MKL_HOME}/include:${CMAKE_INCLUDE_PATH}"
 
+ENV MAKEFLAGS="-j$(nproc)"
+
 RUN source ${VIRTUAL_ENV}/bin/activate \
     && if [ "${INSTRUCTLAB_VERSION}" != "" ] ; then \
         INSTRUCTLAB_PKG="${INSTRUCTLAB_PKG}==${INSTRUCTLAB_VERSION}" ; \
     fi \
-    && pip install torch psutil \
+    && pip install torch==2.4.1 psutil==6.1.0 \
```

Comment:
I believe these are not correct now. Let's update the PR and merge it. I can confirm that the issues listed are still present.